generateStatic.ts

```diff
@@ -1,8 +1,8 @@
-const fs = require("fs");
-const globby = require("globby");
-const path = require("path");
+import * as fs from "fs";
```

Comment:
I think you changed the wrong file ;-p. Keep this file as it was and change generateSitemap.js instead